src/core/sys/linux/sys/auxv.d

```diff
@@ -0,0 +1,165 @@
+/**
+ * D header file for GNU/Linux.
+ *
+ * License:   $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost License 1.0)
+ * Authors:   Kai Nacke
+ */
+module core.sys.linux.sys.auxv;
+
+import core.sys.posix.sys.types;
+
+version (linux):
+extern (C):
+@system:
+nothrow:
+
+c_ulong getauxval(c_ulong type);
```

Comment:
I suspect that `getauxval()` can be `pure`:
http://man7.org/linux/man-pages/man3/getauxval.3.html